stream_alert/rule_processor/config.py

```diff
@@ -33,15 +35,29 @@ def load_config(conf_dir='conf/'):
     key denotes the name of the log type, and includes 'keys' used to match
     rules to log fields.
     """
-    conf_files = ('sources', 'logs', 'types', 'global')
+    conf_files = ('sources', 'logs', 'types', 'global', 'clusters')
     config = dict()
     for base_name in conf_files:
-        path = '{}.json'.format(os.path.join(conf_dir, base_name))
-        with open(path) as data:
-            try:
-                config[base_name] = json.load(data, object_pairs_hook=OrderedDict)
-            except ValueError:
-                raise ConfigError('Invalid JSON format for {}.json'.format(base_name))
+        if base_name == 'clusters':
+            # Load current cluster config into memory for threat intel
+            path = os.path.join(conf_dir, base_name)
+            config['clusters'] = {}
```

Comment:
this could be loaded into a `cluster` key instead since we load a singular file

Reply:
I just want to clarify that do you mean to have key order 
1. `config['prod']` when the current cluster name is `prod`? 
2. Or, `config['cluster']['prod']`?

Reply:
I would prefer to keep `config['clusters']` style, because it is inline with directory structure in `conf/`

Reply:
sounds good @chunyong-lin, feel free to leave then